doc/api/cli.md

```diff
@@ -154,53 +208,80 @@ Note: v8 options allow words to be separated by both dashes (`-`) or underscores
 For example, `--stack-trace-limit` is equivalent to `--stack_trace_limit`.
 
 ### `--tls-cipher-list=list`
+<!-- YAML
+added: v4.0.0
```

Comment:
It looks like that may have been in 0.12 or whatever, but it wasn't in any of the io.js releases, so... I don't know, I think it reduces confusion if we say 4.0.0. Otherwise people need the historic knowledge that 1.0.0-3.x.x are special. I'd prefer to assume people don't know that.